pkg/action/install.go

```diff
@@ -105,15 +106,16 @@ type Install struct {
 
 // ChartPathOptions captures common options used for controlling chart paths
 type ChartPathOptions struct {
-	CaFile   string // --ca-file
-	CertFile string // --cert-file
-	KeyFile  string // --key-file
-	Keyring  string // --keyring
-	Password string // --password
-	RepoURL  string // --repo
-	Username string // --username
-	Verify   bool   // --verify
-	Version  string // --version
+	CaFile     string // --ca-file
+	CertFile   string // --cert-file
+	KeyFile    string // --key-file
+	Keyring    string // --keyring
+	Password   string // --password
+	RepoURL    string // --repo
+	Username   string // --username
+	Verify     bool   // --verify
+	Version    string // --version
+	ReadOnlyFS bool   // --use-inmemory-cache
```

Comment:
We could implement this even without introducing the new field, if we establish the convention that `settings.RepositoryCache` is `nil` or `empty`. IMHO, it makes more sense.

Reply:
I don't know if settings.RepositoryCache can be empty because lazypath sets this by default AFAIK.

Reply:
If I read it correctly here: https://github.com/helm/helm/blob/14f6d1ea97eeef158adf7db0e9b42206905930bf/pkg/cli/environment.go#L95 , an env var could be defined in empty, and this value will used.

Reply:
With this though the behavior will be implicit and not explicit right ?

Reply:
I think for now let's try to keep it explicit functionality. @marckhouzam WDYT ?

Reply:
Well, it depends :) `HELM_REPOSITORY_CACHE` env var is there so that user can customize the cache location. Now, we can set it to empty and with that turn off cache fully, or think about some other special value (e.g. `off`, or `none`) and use it later in the code.

Reply:
I think having a specialty value like "off" or "none" is the best solution. Allows us to to reuse the current flag and configuration.

Reply:
I really like the idea of setting HELM_REPOSITORY_CACHE to "" and avoiding the extra field.  Thanks @pedjak for the idea!

I don't know if there would be side-effects in other areas using the cache, but ultimately, the goal is to not write anything to the file-system, so the cache should never be used.

@waveywaves can you try this approach and see how it goes?

Reply:
@marckhouzam I will try this out.